Tools/WinMLRunner/src/CommandLineArgs.cpp

```diff
@@ -34,6 +34,11 @@ void CommandLineArgs::PrintUsage() {
     std::cout << "  -Debug: print trace logs" << std::endl;
     std::cout << "  -Terse: Terse Mode (suppresses repetitive console output)" << std::endl;
     std::cout << "  -AutoScale <interpolationMode>: Enable image autoscaling and set the interpolation mode [Nearest, Linear, Cubic, Fant]" << std::endl;
+    std::cout << std::endl;
+    std::cout << "Concurrency Options:" << std::endl;
```

Comment:
Can we specify the usage of concurrency options? Will they ever interact with the normal options? Or is ConcurrentLoad a special case where it shouldn't bind / create session / evaluate? In this case we should specify in the -ConcurrentLoad usage.

Up top on line 12 we already specify: 
WinmlRunner.exe <-model | -folder> <fully qualified path> [options]

Reply:
I think in the subsequent changes we should also create sesison/bind/evaluate with concurrency options. For now this change is meant to be a boilerplate code for future additions of concurrency options.

Reply:
Okay sounds good!